<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>751</width>
      <height>411</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Processos</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>1</x>
        <y>11</y>
        <width>729</width>
        <height>21</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>PID</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textEditPID"/>
       </item>
       <item>
        <widget class="QPushButton" name="botaoSuspender">
         <property name="text">
          <string>Suspender</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="botaoContinuar">
         <property name="text">
          <string>Continuar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="botaoMatar">
         <property name="text">
          <string>Matar</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>50</y>
        <width>241</width>
        <height>25</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>CPU</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxCPU">
         <item>
          <property name="text">
           <string>1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="botaoAlterarCPU">
         <property name="text">
          <string>Alterar CPU</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>50</y>
        <width>361</width>
        <height>25</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Prioridade</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxPrioridade">
         <item>
          <property name="text">
           <string>1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>5</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="botaoMudarPrioridade">
         <property name="text">
          <string>Mudar Prioridade</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QTableWidget" name="tabela">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>80</y>
        <width>741</width>
        <height>281</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>PID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>PRIORIDADE</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>USER</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>STATUS</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>#CPU</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>%CPU</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>%MEMORIA</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>0</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>GERENCIADOR DE PROCESSOS</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>750</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>botaoSuspender</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>suspender()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>568</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel">
     <x>555</x>
     <y>34</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>botaoContinuar</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>continuar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>649</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel">
     <x>666</x>
     <y>37</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>botaoMatar</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>matar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>730</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel">
     <x>733</x>
     <y>34</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>botaoMudarPrioridade</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>mudarPrioridade()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>305</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>266</x>
     <y>30</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>botaoAlterarCPU</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>alterarCPU()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>644</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>662</x>
     <y>27</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tabela</sender>
   <signal>cellClicked(int,int)</signal>
   <receiver>MainWindow</receiver>
   <slot>pegarPIDTabela()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>185</x>
     <y>164</y>
    </hint>
    <hint type="destinationlabel">
     <x>159</x>
     <y>37</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>suspender()</slot>
  <slot>matar()</slot>
  <slot>continuar()</slot>
  <slot>mudarPrioridade()</slot>
  <slot>alterarCPU()</slot>
  <slot>pegarPIDTabela()</slot>
 </slots>
</ui>
